defi-pulse/src/data-providers/kaiko.ts

```diff
@@ -0,0 +1,44 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Index } from '../adapter'
+import { util } from '@chainlink/ea-bootstrap'
+
+const getPriceData = async (symbol: string, currency: string) => {
+  const url = `https://us.market-api.kaiko.io/v2/data/trades.v1/spot_exchange_rate/${symbol.toLowerCase()}/${currency.toLowerCase()}`
+  const params = {
+    sort: 'desc',
+  }
+  const headers = {
+    'X-Api-Key': util.getRequiredEnv('API_KEY'),
```

Comment:
Yes, these `API_KEY`s are also too deep and give us little flexibility, but just leave this as is because we plan to refactor this and use the adapters directly in the future.